features/FxA-57-verify-reminder/README.md

```diff
@@ -0,0 +1,44 @@
+
+Verification Reminder Email Specification
+======================
+
+https://mozilla.aha.io/features/FXA-57
+
+As a new Firefox Accounts user,
+ I have signed up for an FxA account when **setting up sync**, but did not verify my email.
+ 
+ I should get 2 email reminders that will ask me to verify my email.
+ 
+ The first email arrives after 48 hours since sign up.
+ 
+ The second email arrives after 168 hours since sign up.
+ 
+## Basic Email Template
+
+> This is just a visual guide, please see detailed email content below
+
+![](basic-template.png)
+
+
```

Comment:
@mozmatej you can see the contents of emails by viewing: https://github.com/mozilla/fxa/pull/70/files